app/src/main/java/org/oppia/android/app/fragment/BottomSheetFragment.kt

```diff
@@ -0,0 +1,35 @@
+package org.oppia.android.app.fragment
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.oppia.android.R
+import org.oppia.android.app.model.ReadingTextSize
+import org.oppia.android.app.options.OptionsActivity
+import org.oppia.android.app.player.exploration.BottomSheetItemClickListener
+import org.oppia.android.app.player.exploration.ExplorationActivity
+import org.oppia.android.databinding.FragmentBottomSheetBinding
+
+class BottomSheetFragment(val internalProfileId : Int) : BottomSheetDialogFragment() {
+
+  override fun onCreateView(
+    inflater: LayoutInflater, container: ViewGroup?,
+    savedInstanceState: Bundle?
+  ): View? {
+    val binding = FragmentBottomSheetBinding.inflate(inflater,container,false)
+    setUpOnClickListeners(binding)
+    return binding.root
+  }
+
+  private fun setUpOnClickListeners(binding: FragmentBottomSheetBinding){
+    val bottomSheetItemClickListener = activity as BottomSheetItemClickListener
+    binding.actionHelp.setOnClickListener {
+      bottomSheetItemClickListener.handleOnOptionsItemSelected(R.id.action_help)
```

Comment:
I think once the click is captured, you should dismiss the dialog fragment. Same applies for other click too. This should mostly solve your error.

Reply:
@rt4914 I tried calling `dismiss()` both begore and after calling `handleOnOptionsItemSelected` but it still gives the same error. I feel the error is not of dismiss because the app does not even compile.

Reply:
Resolving this conversation as the error is solved